client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/SymptomsFields/SymptomsFields.tsx

```diff
@@ -85,6 +90,9 @@ const SymptomsFields: React.FC<Props> = (props: Props): JSX.Element => {
                                                 disabled={wasInvestigationReopend}
                                                 onBlur={props.onBlur}
                                                 maxDate={new Date()}
+                                                minDate={getMinimalSymptomsStartDate(validationDate)}
+                                                maxDateMessage={''}
+                                                minDateMessage={''}
```

Comment:
why add empty fields? (and if they need to be added why the curly braces)

Reply:
I will remove the curly braces, but I need this to override the default mui message